### DISCUSSION TITLE
Osu Unexplained Error

### DISCUSSION THREAD

**Discussion by SLDplays** [ppy/osu#23454] [Q&A]
Keep seeing a " Osu Has Encountered a unexplained error." It doesnt crash or anything but it just bugs out in the backend. Ive attached my entire set of logs
[database.log](https://github.com/ppy/osu/files/11438053/database.log)
[input.log](https://github.com/ppy/osu/files/11438054/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11438055/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/11438056/network.log)
[performance.log](https://github.com/ppy/osu/files/11438057/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/11438058/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/11438059/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/11438060/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/11438061/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/11438062/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11438063/updater.log)

**Comment by bdach** [comment]
> It doesnt crash or anything but it just bugs out in the backend

What does this mean?

> Keep seeing a " Osu Has Encountered a unexplained error."

What is producing this error message, exactly? The logs look clean.
